### ISSUE TITLE
Generating a configuration file Question

### ISSUE THREAD

**Issue by thomasnguyen** [EddyVerbruggen/cordova-plugin-googleplus#593]
Not an issue but a question since I don't know where to ask for help. 

I'm trying to set this up for my Ionic4 build. In the Google API set-up, there are 3 links to generate a configuration file but they all redirect me to a firebase page. Is this an error or is firebase required for this plugin? 

Also, lets just say I am not concerned about iOS, is REVERSED_CLIENT_ID necessary? 

Thank you.

**Comment by Bramzor** [comment]
What I did was go to https://console.developers.google.com/apis/dashboard , select the API that you need and when you did that, go to Credentials and there you can generate your OAuth clientID and secrets for web / iOS and android for that specific API.

Not sure about your second question as I only did use it on iOS.

**Comment by t2wu** [comment]
Yes, why is instruction for REVERSED_CLIENT_ID getting me to the Firebase page?

**Comment by matthewgertner** [comment]
I had the same problem. It seems you don't have to generate the config file anymore. The link in the README is simply wrong. I did what @Bramzor suggested and it worked fine. The client ID is listed in the Credentials section of the Google Developer Console. You just have to "reverse" the ID (i.e. change the order of the segments separated by dots) and pass it to the plugin as `REVERSE_CLIENT_ID` as per the documentation.
